sources/osg/RenderStage.js

```diff
@@ -37,6 +30,41 @@ RenderStage.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit(
     // temporary, Utils.createPrototypeClass will solve this
     constructor: RenderStage,
 
+    cloneType: function () {
+        return this.constructor.prototype.getOrCreate().init();
```

Comment:
I'm ok with that, but that means changing the current cullvisitor::_rootRenderStage mechanism used by application providiing their own RenderStage ?